misago/__init__.py

```diff
@@ -1,2 +1,5 @@
+from .plugins.pluginlist import load_plugin_list_if_exists
```

Comment:
Does this need to be namespaced at the top package level? Should it be under `misago.plugins` instead?

Reply:
I've did this that way so `settings.py` in `misago-docker` can import this via `from misago import load_plugin_list_if_exists`.

Reply:
Got it, makes sense because it is a top-level operation. I'm always just a bit concerned about having things in appropriate namespaces 😁 haha